pkg/resources/model/system.go

```diff
@@ -338,6 +339,7 @@ func FlowForClusterFlow(flow v1beta1.ClusterFlow, clusterOutputs ClusterOutputs,
 					ContainerNames:  match.ClusterExclude.ContainerNames,
 					Hosts:           match.ClusterExclude.Hosts,
 					Namespaces:      match.ClusterExclude.Namespaces,
+					NamespacesRegex: match.ClusterSelect.NamespacesRegex,
```

Comment:
I think this should be 
```suggestion
					NamespacesRegex: match.ClusterExclude.NamespacesRegex,
```

Reply:
Good catch! Thanks @aslafy-z 👍